### PR TITLE
Release candidate v1.5.1

### PR DESCRIPTION
**Pull request by BarryPiccinni**
Add updated declaration content
Add new "other type of injury" labels to PI injuries.
Fix character count error message for p-applicant-affect-on-daily-life-dmi
Bump version to 1.6.1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
     type: 'apply-for-compensation',
-    version: '1.5.0',
+    version: '1.6.1',
```
Comment (armoj) [inline index.js:5]: what info am I missing here? why is there 2 bumps?

File: index.js [hunk 2]
```diff
@@ -3088,7 +3092,7 @@ module.exports = {
                         'This helps us understand how the crime has affected you. You can leave this blank, but we may have to ask for more information later.',
                     maxLength: 1000,
                     errorMessage: {
-                        maxLength: 'Description must be 500 characters or less'
+                        maxLength: 'Description must be 1000 characters or less'
```
Comment (armoj) [inline index.js:3095]: thank you

File: index.js [hunk 3]
```diff
@@ -5650,10 +5654,10 @@ module.exports = {
                 },
                 'q-applicant-physical-injuries-arms-shoulder-other': {
                     type: 'string',
-                    title: 'Type of injury',
+                    title: 'Other shoulder injuries',
```
Comment (armoj) [inline index.js:5657]: this question and the torso-shoulder question have the same descriptors (6 in total). Is the descriptor `Other shoulder injuries` OK for both questions?

File: index.js [hunk 4]
```diff
@@ -2964,10 +2968,10 @@ module.exports = {
                 },
                 'q-applicant-other-treatment-dmi': {
                     type: 'string',
-                    title: 'Other treatments',
+                    title: 'Other mental health treatment',
```
Comment (armoj) [inline index.js:2971]: Line `3007` has not been changed in-line with all other error messages so far. Should line `3007` be changed from: 
`'Enter any other treatment you have received for your mental injury'`
to:
`'Enter any other mental health treatment you have received for your mental injury'`?

**Review by armoj — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (BarryPiccinni) [reply index.js:5]: As confirmed in the chat, this is the correct version!

File: index.js (hunk 3 above)
Comment (BarryPiccinni) [reply index.js:5653]: I'll flag this with Julie but this content is in line with the data dictionary.

File: index.js (hunk 4 above)
Comment (BarryPiccinni) [reply index.js:2967]: Again, this was treated separately from the physical injuries and this content is in line with the data dictionary.

**Review by armoj — APPROVED** [review]

**Pull request merged**